solana/pyth2wormhole/client/Cargo.toml

```diff
@@ -39,6 +39,5 @@ warp = "0.3.3"
 http = "0.2.8"
 
 [dev-dependencies]
-pyth-client = "0.5.0"
```

Comment:
This is wrong! It is used in the standalone tests. I think there should be a way to tell clippy to take `--test` into account.

Reply:
I think our Cargo caching may be giving us a false positive about this.

Reply:
example: test_attest.rs

Reply:
That one I removed manually, sorry that I forgot to mention it myself.
Clippy was complaining that the package is deprecated (we marked all its method deprecated). I replaced it with pyth_sdk_solana